hatchet_sdk/utils/serialization.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any
+
+
+def flatten(xs: dict[str, Any], parent_key: str, separator: str) -> dict[str, Any]:
```

Comment:
Python annoyance: In general I try to avoid using `Any`, but type hinting for arbitrarily nested dictionaries is really clunky AFAIK